ignite-base/App/Containers/App.js

```diff
@@ -1,14 +1,11 @@
 // @flow
```

Comment:
We're loading `../Config` at the top because load order matters and we want to make sure all the config has a chance to be applied before we load anything else.